app/src/main/java/io/homeassistant/companion/android/controls/MediaPlayerControl.kt

```diff
@@ -0,0 +1,98 @@
+package io.homeassistant.companion.android.controls
+
+import android.content.Context
+import android.os.Build
+import android.service.controls.Control
+import android.service.controls.DeviceTypes
+import android.service.controls.actions.BooleanAction
+import android.service.controls.actions.ControlAction
+import android.service.controls.actions.FloatAction
+import android.service.controls.templates.ControlButton
+import android.service.controls.templates.RangeTemplate
+import android.service.controls.templates.ToggleRangeTemplate
+import android.service.controls.templates.ToggleTemplate
+import androidx.annotation.RequiresApi
+import io.homeassistant.companion.android.common.R as commonR
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.common.data.integration.IntegrationRepository
+import io.homeassistant.companion.android.common.data.integration.getVolumeLevel
+import io.homeassistant.companion.android.common.data.integration.getVolumeStep
+import io.homeassistant.companion.android.common.data.integration.supportsVolumeSet
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+@RequiresApi(Build.VERSION_CODES.R)
+object MediaPlayerControl : HaControl {
+    override fun provideControlFeatures(
+        context: Context,
+        control: Control.StatefulBuilder,
+        entity: Entity<Map<String, Any>>,
+        info: HaControlInfo
+    ): Control.StatefulBuilder {
+        val isPlaying = entity.state == "playing"
```

Comment:
Side effect I noticed while testing: now it visually turns off when buffering. Should we match the frontend here instead, even if the toggle action makes less sense? <https://github.com/home-assistant/frontend/blob/e58bef7795f610b40820f5b8df5e62d733fe4642/src/common/entity/state_active.ts#L42> active = not `standby`.

Reply:
i originally had this but changed it based on internal discussion :)

Reply:
Seeing it live cam change your mind :)